packages/legend-art/style/base/_common.scss

```diff
@@ -77,6 +77,19 @@
   margin: 0;
 }
 
+.dnd__entry__handle__container {
+  float: left;
+  cursor: move;
+  // we have relative position so that z-index will apply
```

Comment:
TODO: move this down to where we have the placeholder CSS class

the comment here should be more like: we need to ensure drag handle needs to be at least above placeholder in the stacking context, else while dragging and the placeholder is displayed, we would lose the handle